app/views/partials/foyer/enfants.html

```diff
@@ -13,7 +13,8 @@
           <div class="panel-body">
             {{ enfant.firstName }}
             <button type="button" class="close" ng-click="removeEnfant(enfant)">
-              <span aria-hidden="true">&times;</span>
+              <span class="screenReadersOnly">Effacer</span>
+              <span aria-role="presentation" aria-hidden="true">&times;</span>
```

Comment:
To be honest, I don't know exactly. I used to think `role="presentation"` was enough, but [it seems not](http://john.foliot.ca/aria-hidden/).